vss-graphql/src/index.ts

```diff
@@ -11,19 +11,47 @@ const runServer = async (): Promise<void> => {
     port = Number.isNaN(parsedPort) ? defaultPort : parsedPort;
   }
 
-  const low = require('lowdb');
-  const FileSync = require('lowdb/adapters/FileSync');
-
-  const adapter = new FileSync('/usr/shared/db/db.json');
-  const db = low(adapter);
+  const sqlite3 = require("sqlite3");
+  const dbPath = "/usr/shared/db/opends.db";
+  let sqlite_db = new sqlite3.Database(dbPath, (err) => {
+    if (err) {
+      return console.error(err.message);
+    }
+    console.log('Connected to the opends database.');
+  });
 
   const resolvers = {
     Query: {
       vehicle(parent, args, context, info) {
-        db.read();
-        const veh = db.get('vehicle').value()
-        console.log(JSON.stringify(veh));
-        return veh;
+        let key = "";
+        let object = new Object();
+        let root = new Object();
+
+        let promise = new Promise((resolve, reject) => {
+          sqlite_db.each("SELECT key, value FROM vss_data", (err, row) => {
+            key = row.key;
+            let leaves = key.split('.');
```

Comment:
you could use lodash `.set()` to do this in a simpler way: https://lodash.com/docs/4.17.15#set

Reply:
Suggestion implemented. Thanks.